packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep3.tsx

```diff
@@ -0,0 +1,38 @@
+import { Warning, PassphraseTypeCard } from '@trezor/components';
+import { Translation } from 'src/components/suite/Translation';
+import { PassphraseHeading } from './PassphraseHeading';
+import { OpenGuideFromTooltip } from 'src/components/guide';
+
+type PassphraseWalletConfirmationStep3Props = {
+    onDeviceOffer: boolean;
+    onSubmit: (value: string, passphraseOnDevice?: boolean) => void;
+};
+
+export const PassphraseWalletConfirmationStep3 = ({
+    onDeviceOffer,
+    onSubmit,
+}: PassphraseWalletConfirmationStep3Props) => {
+    return (
```

Comment:
nit: inline return `=> (<> ....)` with no `{` and no `return`

I thought we have a lint for this. :thinking: